yapapi/executor/ctx.py

```diff
@@ -57,8 +60,14 @@ def register(self, commands: CommandContainer):
 
 
 class _Start(Work):
+    def __init__(self, *args: str):
+        self.args = args
+
+    def __repr__(self):
+        return f"start{self.args}"
```

Comment:
Should this be concatenated?

Reply:
looks okay, will print  e.g. "start('arg1', 'arg2')" (`self.args` is a tuple)

Reply:
@zakaprov indeed, as @azawlocki stated